artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQMessageBundle.java

```diff
@@ -559,4 +560,6 @@ IllegalStateException invalidRoutingTypeUpdate(String queueName,
    @Message(id = 229255, value = "Bridge {} cannot be {}. Current state: {}")
    ActiveMQIllegalStateException bridgeOperationCannotBeExecuted(String bridgeName, String failedOp, BridgeImpl.State currentState);
 
+   @Message(id = 229256, value = "Invalid address settings {} for page store {}. Details: {}")
+   ActiveMQException invalidPageSettings(String pageStoreName, AddressSettings addressSettings, String details);
```

Comment:
This new exception doesnt appear to be getting thrown anywhere

Reply:
good catch. I forgot to delete it. :)

Reply:
Perhaps, but it seemed more like it should be getting used hehe, e.g in the constructor scenario from my other comment :)

Reply:
ah you mean in constructor scenario we can throw an exception instead of log a warning? It seems makes sense.
I'll add it back.